### PR TITLE
Ra 81 - Mainly PlayerObject

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: VAS.Drawing.Cairo/CairoBackend.cs [hunk 1]
```diff
@@ -28,6 +28,7 @@
 using Image = VAS.Core.Common.Image;
 using LineStyle = VAS.Core.Common.LineStyle;
 using Point = VAS.Core.Common.Point;
+using VAS.Core;
```
Comment (ylatuya) [inline VAS.Drawing.Cairo/CairoBackend.cs]: Sort using

File: VAS.Drawing.Cairo/CairoBackend.cs [hunk 2]
```diff
@@ -154,7 +155,7 @@ public CairoBackend ()
 
 		public ISurface CreateSurface (string filename, bool warnOnDispose = true)
 		{
-			Image img = new Image (filename);
+			Image img = Resources.LoadImage (filename);
```
Comment (ylatuya) [inline VAS.Drawing.Cairo/CairoBackend.cs]: Image uses a full path to the image while LoadImage uses a relative path.

Comment (vguzmanp) [inline VAS.Drawing.Cairo/CairoBackend.cs]: Hm... I changed this because something was calling this method with a relative path. Let me check again

Comment (vguzmanp) [inline VAS.Drawing.Cairo/CairoBackend.cs]: I see now, but anyways Resources.LoadImage should be safe: `Path.Combine` with an absolute path returns the same absolute path.

**Comment by ylatuya** [comment]
👍

**Pull request merged**